src/meta/processors/BaseProcessor.inl

```diff
@@ -236,41 +230,112 @@ StatusOr<EdgeType> BaseProcessor<RESP>::getEdgeType(GraphSpaceID spaceId,
     return Status::EdgeNotFound(folly::stringPrintf("Edge %s not found", name.c_str()));
 }
 
+template<typename RESP>
+StatusOr<std::vector<std::string>>
+BaseProcessor<RESP>::getLatestTagFields(GraphSpaceID spaceId,
```

Comment:
In theory, we could support multiple versions of a single index. But in reality, I doubt how useful it is. Any change to the index would invalidate all existing index data. So I would rather to create a new index and to drop the old one

Reply:
Yes, it's true. When we design the index we have discussed multi versions index. It look like equal with drop the index and recreation it (rebuild index maybe a big task). So I my option, it's unnecessary.